setup/schemas.sql

```diff
@@ -0,0 +1,77 @@
+USE library_management_system;
+
+IF NOT EXISTS
+    (
+        SELECT TOP 1 1
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE TABLE_SCHEMA = 'dbo'
+            AND TABLE_NAME = 'book'
+            AND COLUMN_NAME = 'book_id'
+    )
+BEGIN
+    CREATE TABLE book (
+    book_id INT PRIMARY KEY IDENTITY (1, 1),
+    title NVARCHAR(100) NOT NULL,
+    author NVARCHAR(100) NOT NULL,
+    isbn NVARCHAR(17) NOT NULL,
+    genre NVARCHAR(50) NOT NULL,
+    shelf_location NVARCHAR(100) NOT NULL,
+    status BIT DEFAULT 0 NOT NULL,
```

Comment:
when name a boolean is the database you should name it "is{something}", to make it clear that this is a bit / boolean and it will be more clearer to know what it represent

so for your case naming it "is_available" or "is_borrowed" will be more clear than "status"